### ISSUE TITLE
[FEATURE REQUEST] Add another GRID source: useful for RV, boat, also homes. FIrst Grid is regular power from grid, second "grid" is used for local (diesel) Generator

### ISSUE THREAD

**Issue by Klagio** [flixlix/flixlix-cards#211]
Generator are used in many application, they usually turn on when Grid is disconnected (power cut in a home, or in case of RV and boat when they are away from shore)

**Comment by philspain72** [comment]
Was going to add very similar comment a Generator would be great

**Comment by nosek-tomas** [comment]
I think it would be useful to have it configurable...

 Have, for example, a "subspecies" of GRID.  I wanted to use something similar to display "Virtual Battery"...charge/discharge and show capacity...

 In short, it would be good to have the same individuals as for the house...

**Comment by ubarni** [comment]
Would also appreciated a 2nd "Grid" source. All RVs have grid AND alternator power supply.

**Comment by tomsride** [comment]
YES, please! That would be a great addition, Thank for your great work!
